components/feature/awesomebar/src/main/java/mozilla/components/feature/awesomebar/AwesomeBarFeature.kt

```diff
@@ -50,10 +51,11 @@ class AwesomeBarFeature(
      * Add a [AwesomeBar.SuggestionProvider] for "Open tabs" to the [AwesomeBar].
      */
     fun addSessionProvider(
+        resources: Resources,
         sessionManager: SessionManager,
         selectTabUseCase: TabsUseCases.SelectTabUseCase
     ): AwesomeBarFeature {
-        val provider = SessionSuggestionProvider(sessionManager, selectTabUseCase, icons)
+        val provider = SessionSuggestionProvider(resources, sessionManager, selectTabUseCase, icons)
```

Comment:
nit: The change here and in `SessionSuggestionProvider` change the public API. We may want to mention that in the changelog.